### PR TITLE
Geneva agent integration

### PR DESCRIPTION
**Pull request by rafsalas19**
- Add publisher script
- Add publisher config
- Add dependencies for publisher to install scripts
- Add functionality to deploy and shutdown publisher

### CODE REVIEW

File: src/worker/install/install.sh [hunk 1]
```diff
@@ -1,7 +1,16 @@
...
-
 arch="nvidia"
 
+PUBLISHER_INSTALL=$1
+
+if [ -n "$PUBLISHER_INSTALL" ];
+then
+    if [ $PUBLISHER_INSTALL = true ];
+    then
+        $(dirname "${BASH_SOURCE[0]}")/geneva.sh  $(dirname "${BASH_SOURCE[0]}")/config/geneva_config.json
```
Comment (RyoYang) [inline src/worker/install/install.sh:10]: do we need to add `sudo` here as we will rm and create soem dir/file under /etc ?

**Review by RyoYang — APPROVED** [review]

**Comment by RyoYang** [comment]
I also encountered the dcgm version issue which caused nv-exporter processes can not be launched, fixed by installing latest dcgm by `sudo apt-get update \
&& sudo apt-get install -y datacenter-gpu-manager`

File: docs/GenevaAgent.MD [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+6. 
+
+Steps
+-----
+1. Ensure that all prequisites are met.
+2. deploy Moneo:
+  - Full deployment ```python3 moneo.py -d full -g -c ~/hostfile --manager_host localhost```
+  - Worker deployment ```python3 moneo.py -d workers -g -c ~/hostfile```
+  Note: if install step has already been performed you can use the -w flag to skip it.
+3. Verify that the exporters andpublisher are running with ```ps -eaf | grep python3```
```
[the remark this answers is not in the document]
Comment (rafsalas19) [reply docs/GenevaAgent.MD]: addressed

File: moneo.py [hunk 3]
```diff
@@ -134,7 +132,13 @@ def deploy_worker(self, hosts_file, max_threads=16, skip_install=False, prof_met
         print('-Starting metric exporters on workers-')
         logging.info('Starting metric exporters on workers')
         cmd = '/tmp/moneo-worker/start.sh'
-        if prof_metrics:
+        if self.args.profiler_metrics:
+            logging.info('Profiling enabled')
+            cmd = cmd + ' true'
+        else:
+            cmd = cmd + ' false'
```
[the remark this answers is not in the document]
Comment (rafsalas19) [reply moneo.py:144]: Addressed

**Pull request merged**